BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetPersonBySearchTermQueryHandlerEFCoreTests.cs

```diff
@@ -0,0 +1,92 @@
+using System.Linq;
+using System.Threading.Tasks;
+using BookingsApi.DAL;
+using BookingsApi.DAL.Queries;
+using BookingsApi.Domain;
+using BookingsApi.Domain.Participants;
+using BookingsApi.Domain.RefData;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+
+namespace BookingsApi.IntegrationTests.Database.Queries
+{
+    public class GetPersonBySearchTermQueryHandlerEFCoreTests
+    {
+        private GetPersonBySearchTermQueryHandler _handler;
+        private BookingsDbContext _context;
+        private Person Person1, Person2, Person3;
```

Comment:
One minor thing, rename personX and participantX to something like JudgePerson, JudgeParticipant. It would make reading the tests a lot easier.